packages/elements-angular/package.json

```diff
@@ -24,52 +24,51 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~10.1.4",
-    "@angular/common": "~10.1.4",
-    "@angular/compiler": "~10.1.4",
-    "@angular/core": "~10.1.4",
-    "@angular/forms": "~10.1.4",
-    "@angular/platform-browser": "~10.1.4",
-    "@angular/platform-browser-dynamic": "~10.1.4",
-    "@angular/router": "~10.1.4",
+    "@angular/animations": "~12.0.1",
```

Comment:
I would recommend using `^12.0.1` using for all dependencies to avoid warning on angular 13 etc. Hopefully it works out of the box.